src/index.ts

```diff
@@ -27,30 +25,31 @@ export function _buildMetadataAccessor(type: string) {
       property = '/' + options.property;
     }
 
-    const reqOpts = extend(
-                        true, {
-                          uri: BASE_URL + '/' + type + property,
+    const reqOpts = Object.assign(
+                        {
```

Comment:
`extend(true, ..)` was a deep clone, but `Object.assign` will be a shallow clone. Is this change intentional, i.e. is there rationale for why a deep clone was unnecessary?

Reply:
I keep forgetting about `Object.assign` not supporting deep clone :|  I put it back.